packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -194,12 +194,17 @@ export const CallComposite = (props: CallCompositeProps): JSX.Element => {
   } = props;
   useEffect(() => {
     (async () => {
-      await adapter.askDevicePermission({ video: true, audio: true });
-      adapter.queryCameras();
-      adapter.queryMicrophones();
+      if (role === 'Consumer') {
+        // Need to ask for audio devices to get access to speakers
```

Comment:
Do we need this? I don't think access to speaker need any permissions? I don't remember youtube is asking me any permissions to play a video

Reply:
I will look into how to get speakers access without permissions from the Calling SDK.